### DISCUSSION TITLE
Dark Theme too different from Twidere

### DISCUSSION THREAD

**Discussion by yoshimo** [TwidereProject/TwidereX-Android#47] [General]
It may be the font setting, it might be the rough first release, but i have my issues with the current look of the new version.
Coming from the original twidere it feels kind of wrong.
I hope you will go for something more similiar with the next releases.

**Comment by yoshimo** [comment]
![Twidere](https://user-images.githubusercontent.com/17909/101291927-937ae580-380c-11eb-93af-b3c52728c6cf.jpg)
![TwidereX](https://user-images.githubusercontent.com/17909/101291930-97a70300-380c-11eb-8da7-dd7444a5c5b1.jpg)

**Comment by yoshimo** [comment]
Comparing those i notice the following
Twidere looks a bit brighter.

TwidereX abbreviates texts instead of showing all lines
TwidereX shows Twitter shortlinks t.co instead of what is behind it.
TwidereX has trouble with encoding "&amp" 
I miss the old "fat" icons  for answer, retwet, share and like. The new ones are a bit skinny.

Regardless of the fontsize, Both Nickname and name should be fully written.
It would be nice to have both date&time shown of the post.

The whole profile feels a bit larger, because of the icons used instead of the old text.
The filters are missing for now, but i guess that is part of being the first release.

**Comment by solonovamax** [comment]
Honestly, I MUCH prefer the TwidereX theme. I just found out about Twidere, and the one thing that kind of put me off is how bad (in my opinion) it looked, coming from the twitter app.
But I do love how much customization it has. Which is why I hope TwidereX becomes just as powerful, while looking a whole lot nicer.

(It *would* be nice to have the ability to load custom CSS themes though. Perhaps as extensions? :eyes:)

**Comment by IBBoard** [comment]
Could we have a mid-grey theme like Twidere had? TwidereX only has "bright white" and "so dark black that you get eye strain". Twidere was one of the few apps that got the "dark but not too dark" theme right.
